README.md

```diff
@@ -9,16 +10,60 @@ This repository contains basic set of shared libraries and so-called SCR agents
 used for reading and writing configuration files and some even for executing
 commands on the system.
 
-## Installation ##
 
-    make -f Makefile.cvs
-    make
-    sudo make install
+Provided Functionality
+======================
 
-## Running Testsuites ##
+See [the generated yardoc documentation](http://www.rubydoc.info/github/yast/yast-yast2) at rubydoc.info.
 
-    make check
+Development
+===========
 
-## Links ##
+This module is developed as part of YaST. See the generic
+[development documentation](doc/README_Generic.md#yast-development-documentation) and
+[development environment](doc/README_Generic.md#development-environment) description.
```

Comment:
These two links point to the same page and almost at the same point. It's confusing. I'd just keep one.